quotes/static/quotes/css/quotes.css

```diff
@@ -0,0 +1,84 @@
+
+.profilepage__thumbwrapper {
+    margin-top: 3em;
+}
+
+.profilepage__position {
+    margin-top: 2em;
+    margin-bottom: 0.25em;
+}
+
+.profilepage__ksginfo > p {
+    margin: 0.3em 0;
+}
+
+.quote__text {
+    font-style: italic;
+}
+
+.no-bullet {
+    list-style-type: none;
+}
+
+.quote_box {
+
+}
+
+.grid-container {
```

Comment:
These should probably be named differently. We now risk that there are multiple "grid-container" and "grid-item" classes, interfering with one-another.

My suggestion is to use [BEM](http://getbem.com/naming/) here. The class names can, for instance, be `quotes__gridcontainer` and `quotes__griditem`.